### DISCUSSION TITLE
What term describes the ethical theory that . . .

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#71] [Galaxy Qs] [answered]
Within the realm of philosophy, what term describes the ethical theory that argues moral judgments are subject to the cultural context and time period in which they arise, denying universal moral truths?

**Comment by Denisyasyuchenya** [comment]
The ethical theory that argues moral judgments are subject to the cultural context and time period in which they arise, thereby denying universal moral truths, is known as "Ethical Relativism." This theory holds that there are no absolute moral values that apply to all cultures and times; instead, what is morally right or wrong depends on the beliefs, customs, and practices of a particular society.

**Reply by Sriansh-raj** [reply]
Ethical relativism is the theory that holds that morality is relative to the norms of one's culture. so here the answer will be `Cultural relativism`

Well @Denisyasyuchenya you are just one correct answer away from getting a badge. Keep trying 👍🏻

**Comment by mitchiemt11** [comment, marked as answer]
Cultural relativism

**Reply by Sriansh-raj** [reply]
Yep it is 👍🏻
